pages/EntryListPage/index.js

```diff
@@ -0,0 +1,21 @@
+import JournalEntryList from "@/components/JournalEntryList";
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function EntryListPage() {
+  return (
+    <>
+      {" "}
+      <Link href="/">
+        <StyledLink>Back</StyledLink>
+      </Link>
+      <JournalEntryList />
+    </>
+  );
+}
+const StyledLink = styled.p`
+  display: center;
+  text-align: center;
+  font-size: 1em;
+  color: blue;
+`;
```

Comment:
You can use styled-components to extend that next Link component

```suggestion
export default function EntryListPage() {
  return (
    <>
      <StyledNextLink href="/">
          Back
      </StyledNextLink>
      <JournalEntryList />
    </>
  );
}
const StyledNextLink = styled(Link)`
  text-align: center;
  font-size: 1em;
  color: blue;
`;
```

Also this is no valid value for display

```css
display: center;

```